_data/committers.yml

```diff
@@ -379,6 +379,10 @@
   role: Committer
   alias: mneumann
   affiliation: InfluxData
+- name: Matthijs Brobbel
+  role: Committer
+  alias: mbrobbel
+  affiliation: Voltron Data
```

Comment:
@mbrobbel  can you please verify this is how your affiliation should be listed?

Reply:
This looks good, thank you!